src/Features/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -37,14 +36,10 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
         new FeatureFlagStorageLocation("Roslyn.DisableBackgroundCompilation"));
 
     /// <summary>
-    /// This option allows the user to enable this. We are putting this behind a feature flag for now since we could have extensions
-    /// surprised by this and we want some time to work through those issues.
+    /// This option allows the user to disable this, in case they have some extension that breaks with this on. The expectation is to remove
+    /// this in 17.6 or 17.7.
     /// </summary>
-    public static readonly Option2<bool?> EnableOpeningSourceGeneratedFilesInWorkspace = new(
-        "WorkspaceConfigurationOptions", "EnableOpeningSourceGeneratedFilesInWorkspace", defaultValue: null,
-        new RoamingProfileStorageLocation("TextEditor.Roslyn.Specific.EnableOpeningSourceGeneratedFilesInWorkspaceExperiment"));
-
-    public static readonly Option2<bool> EnableOpeningSourceGeneratedFilesInWorkspaceFeatureFlag = new(
-        "WorkspaceConfigurationOptions", "EnableOpeningSourceGeneratedFilesInWorkspaceFeatureFlag", WorkspaceConfigurationOptions.Default.EnableOpeningSourceGeneratedFiles,
-        new FeatureFlagStorageLocation("Roslyn.SourceGeneratorsEnableOpeningInWorkspace"));
+    public static readonly Option2<bool> EnableOpeningSourceGeneratedFilesInWorkspace = new(
+        "WorkspaceConfigurationOptions", "EnableOpeningSourceGeneratedFilesInWorkspace", defaultValue: true,
+        new RoamingProfileStorageLocation("TextEditor.Roslyn.Specific.EnableOpeningSourceGeneratedFilesInWorkspace"));
```

Comment:
i don't see a feature-flag anymore.  that def worries me if we're late in the game here.  turning this on by default (when SGs have been a huge source of perf issues across the board for the IDE seems concerning to me).